dataframe-csv/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readDelim.kt

```diff
@@ -228,15 +222,12 @@ public fun DataFrame.Companion.readDelim(
  *
  * [DataFrame.readDelimStr][readDelimStr]`("a,b,c", delimiter = ",")`
  *
- * _**NOTE EXPERIMENTAL**: This is a new set of functions, replacing the old [DataFrame.readDelim][org.jetbrains.kotlinx.dataframe.io.readDelim]`()` functions.
- * They'll hopefully be faster and better._
- *
- * @param file The file to read. Can also be compressed as `.gz` or `.zip`, see [Compression][org.jetbrains.kotlinx.dataframe.io.Compression].
+ * @param file The file to read. Can also be compressed as `.gz` or `.zip`, see [Compression].
```

Comment:
Is it navigable correctly for now?

Reply:
it may not be, good catch! KoDEx is not good at redirecting references/paths outside the current module.